src/Components/ServiceExtensions.cs

```diff
@@ -11,5 +11,15 @@ public static IServiceCollection AddComponentSystem(this IServiceCollection serv
 			services.TryAddTransient<ComponentSystem, StateManagedComponentSystem>();
 			return services;
 		}
+
+        public static IServiceCollection AddInMemoryEventStream(this IServiceCollection services)
+        {
+            var inMemoryInstance = new InMemoryEventStream();
+
+            services.TryAdd(ServiceDescriptor.Instance(typeof(EventStream), inMemoryInstance));
```

Comment:
Why instance instead of singleton and let container resolve if/when it is needed?